docs/docs/reference/core-plugins/elasticsearch-plugin/index.md

```diff
@@ -53,7 +53,7 @@ const config: VendureConfig = {
 ## Search API Extensions
 This plugin extends the default search query of the Shop API, allowing richer querying of your product data.
 
-The [SearchResponse](/reference/graphql-api/admin/object-types/#searchresponse) type is extended with information
+The [SearchResponse](/reference/graphql-api/shop/object-types/#searchresponse) type is extended with information
```

Comment:
This markdown file is actually generated, you'll need to update the source comment here: https://github.com/vendure-ecommerce/vendure/blob/82512eec312edcbea198e96406c3a7e3b239c1b6/packages/elasticsearch-plugin/src/plugin.ts#L100